tests/unit/services/testNumber.test.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/first */
+const mockSend = jest.fn();
+
+import { generateNewNumber, NumberTypes } from '../../../src/services/testNumber';
+
+process.env.TEST_NUMBER_LAMBDA_NAME = 'cvs-svc-local-test-number';
+
+jest.mock('@aws-sdk/client-lambda', () => ({
+  LambdaClient: jest.fn(() => ({
+    send: mockSend,
+  })),
+  InvokeCommand: jest.fn(),
+}));
+
+describe('Test test Number Service', () => {
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...OLD_ENV };
+  });
+
+  afterAll(() => {
+    process.env = OLD_ENV;
+  });
+  describe('Successful response with local environment variable set', () => {
+    it('should return a system number', async () => {
+      process.env.AWS_SAM_LOCAL = 'true';
```

Comment:
These tests aren't actually testing the function right? Because this env var just means it returns on the first conditional rather than actually checking the functionality inside no?

Reply:
yeah something like the test below would be good (with the mocked buffer). Unless you're doing this for coverage. in that case I'd suggest something like
```ts
const testCases = [NumberTypes.SystemNumber, NumberTypes,TrailerId, NumberTypes.ZNumer, ...]

it.each(testCases)('should return $number', (number: NumberTypes) => {
  process.env.AWS_SAM_LOCAL = 'true'
  const result = await generateNewNumber(number)
  expect(result).toBe('123')
}
```

The `it.each` probably would be good for the mocked bufferVersion anyways

Reply:
Never seen that before, that's snazzy! Yes, I was doing it for test coverage. I'll implement something like that above^^